<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!--
 * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 * Copyright 2022 Xyna GmbH, Germany
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *  http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-->
<ExceptionStore xmlns="http://www.gip.com/xyna/3.0/utils/message/storage/1.1" Name="ExceptionStore" Version="1.8">
  <ExceptionType BaseTypeName="XynaExceptionBase" BaseTypePath="core.exception" Code="TESTFACTORY-00122" Label="Test Case still referenced from Test Case Chain" TypeName="TestCaseStillReferencedFromTestCaseChain" TypePath="xdev.xtestfactory.infrastructure.exceptions">
    <Data Label="Test Case Chain Name" VariableName="testCaseChainName">
      <Meta>
        <Type>String</Type>
      </Meta>
    </Data>
    <MessageText Language="EN">Test Case still referenced from Test Case Chain '%0%'</MessageText>
    <MessageText Language="DE">Testfall wird noch von der Testfallkette '%0%' referenziert</MessageText>
  </ExceptionType>
</ExceptionStore>
